### PR TITLE
feat(Task-1C): As a Project Manager, already done exercises uploaded to the git repository

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

File: Ejercicio3/index-START.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    function handleUpdate(){
+      //console.log(this.value);
+      //console.log(this.dataset);
+      const suffix =this.dataset.sizing || '';
+      //console.log(suffix);
+      document.documentElement.style.setProperty(`--${this.name}`,this.value + suffix);
+    }
+
+    inputs.forEach(input => input.addEventListener('change', handleUpdate));
+    inputs.forEach(input => input.addEventListener('mousemove', handleUpdate));
```
Comment (eloydotco) [inline Ejercicio3/index-START.html]: Please avoid to iterate something twice, you can do it with just one forEach :)

File: Ejercicio3/index-START.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    input {
+      width: 100px;
+    }
+  </style>
+
+  <script>
+    const inputs = document.querySelectorAll('.controls input');
+    function handleUpdate(){
+      //console.log(this.value);
+      //console.log(this.dataset);
```
Comment (eloydotco) [inline Ejercicio3/index-START.html]: Please, remove commented lines of code :)

**Review by eloydotco — CHANGES_REQUESTED** [review]

File: Ejercicio3/index-START.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    function handleUpdate(){
+      const suffix =this.dataset.sizing || '';
+      document.documentElement.style.setProperty(`--${this.name}`,this.value + suffix);
+    }
+
+    inputs.forEach(input => input.addEventListener('change', handleUpdate), input => input.addEventListener('mousemove', handleUpdate));
+  </script>
+
+</body>
+</html>
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio3/index-START.html:76]: Change the target branch of this PR to 'development' and check it on your other branches

File: Ejercicio3/index-START.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    input {
+      width: 100px;
+    }
+  </style>
+
+  <script>
+    const inputs = document.querySelectorAll('.controls input');
+    function handleUpdate(){
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio3/index-START.html]: Place a whitespace between '()' and '{'

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

File: Ejercicio3/index-START.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  </style>
+
+  <script>
+    const inputs = document.querySelectorAll('.controls input');
+    function handleUpdate() {
+      const suffix = this.dataset.sizing || '';
+      document.documentElement.style.setProperty(`--${this.name}`,this.value + suffix);
+    }
+
+    inputs.forEach(input => input.addEventListener('change', handleUpdate), input => input.addEventListener('mousemove', handleUpdate));
```
Comment (eloydotco) [inline Ejercicio3/index-START.html]: Did you test this arrow functions?? works?? 😕

**Review by eloydotco — CHANGES_REQUESTED** [review]

File: Ejercicio3/index-START.html (hunk 5 above)
Comment (SamuFigueirido) [reply Ejercicio3/index-START.html]: Yes, it works

Comment (eloydotco) [reply Ejercicio3/index-START.html]: Please, check it again, maybe this helps you :)
We can take a look together if you want 👍 

```
  <script>
    const inputs = document.querySelectorAll('.controls input');
    function handleUpdate(ev) {
      console.log(ev.type);
      const suffix = this.dataset.sizing || '';
      document.documentElement.style.setProperty(`--${this.name}`,this.value + suffix);
    }

    inputs.forEach(input => input.addEventListener('change', handleUpdate), input => input.addEventListener('mousemove', handleUpdate));
  </script>
```

Comment (SamuFigueirido) [reply Ejercicio3/index-START.html]: Ok, sorry about that, I have already changed the code :D

Comment (eloydotco) [reply Ejercicio3/index-START.html]: mmm 😕 
did you check if the mousemove event type appears on the devtools console?? :D

Comment (SamuFigueirido) [reply Ejercicio3/index-START.html]: Yes, sorry, i'm silly
I checked and I found the error. I think it is ok now

**Review by eloydotco — APPROVED** [review]

File: Ejercicio3/index-START.html (hunk 5 above)
Comment (eloydotco) [reply Ejercicio3/index-START.html]: great 👍  don't worry, this happens all the time :)

**Pull request merged**